content/deployment/byok-data-plane-setup-on-generic.md

```diff
@@ -30,67 +30,62 @@ helm repo add unionai https://unionai.github.io/helm-charts/
+ -------------- ------------------------------------ ---------------------------- ------------------------------------------------- ------------------------------------------------------------------ ---------- 
+| xxxxxxxxxxx  | xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx | xxxxxxxxxxxxxxxxxxxxxxxxxx | xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx | xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx | xxxxx    |
+ -------------- ------------------------------------ ---------------------------- ------------------------------------------------- ------------------------------------------------------------------ ---------- 
 1 rows
-```
-* Save the secret that is displayed. Union does not store the credentials and it cannot be retrieved later.
-
-3.  Create a values file that include, at a minimum, the following fields:
-
-```yaml
-host: <YOUR_UNION_CONTROL_PLANE_URL>  # Should not include the `https://`.
-clusterName: <MY_CLUSTER> #unique cluster identifier
-orgName: <MY_ORG> #Name of your {{< key product_name >}} organization
-provider: metal #The cloud provider your cluster is running in.  Acceptable values include `aws`, `gcp`, `azure`, `oci`, and `metal` (for self-managed or on-prem clusters).
-storage:
-  provider: "compat"
-  endpoint: <STORAGE_ENDPOINT> #This is the S3 API endpoint provided by your cloud vendor.
-  accessKey: <S3_ACCESS_KEY>
-  secretKey: <S3_SECRET_KEY>
-  bucketName: <S3_BUCKET_NAME>
-  fastRegistrationBucketName: <S3_BUCKET_NAME> #This can be the same as bucketName
-  region: <CLOUD_REGION> #region where your S3 bucket is configured
-secrets:
-  admin:
-    create: true
-    clientId: <UNION_CLIENT_ID> # Generated in the previous step
-    clientSecret: <UNION_CLIENT_SECRET> # Generated in the previous step
-fluentbit:
-  env:
-    - name: AWS_ACCESS_KEY_ID
-      value: <S3_ACCESS_KEY>        # Use the same <S3_ACCESS_KEY> as in storage section
-    - name: AWS_SECRET_ACCESS_KEY
-      value: <S3_SECRET_KEY>        # Use the same <S3_SECRET_KEY> as in storage section
-```
 
-4. Optionally configure the resource `limits` and `requests` for the different services.  By default these will be set minimally, will vary depending on usage, and follow the Kubernetes `ResourceRequirements` specification.
-    * `clusterresourcesync.resources`
-    * `flytepropeller.resources`
-    * `flytepropellerwebhook.resources`
-    * `operator.resources`
-    * `proxy.resources`
+✅ Generated <ORGNAME>-values.yaml
+======================================================================
+Installation Instructions
+======================================================================
 
-5. Install the {{< key product_name >}} operator and CRDs:
-```shell
-helm upgrade --install unionai-dataplane-crds unionai/dataplane-crds
-helm upgrade --install unionai-dataplane unionai/dataplane \
-    --create-namespace \
+Step 1: Prepare your on-prem Kubernetes cluster.
```

Comment:
I'd change the command output to 

`Step 1. Prepare your Kubernetes cluster.`

All the neo clouds (Nebius, Crusoe, Hypertsack, etc)  fall in the category of "generic" K8s but they are not on prem